packages/devtools_app/lib/src/screens/profiler/cpu_profiler.dart

```diff
@@ -179,6 +179,21 @@ class _CpuProfilerState extends State<CpuProfiler>
               const SizedBox(width: denseSpacing),
               UserTagDropdown(widget.controller),
               const SizedBox(width: denseSpacing),
+              ValueListenableBuilder<bool>(
+                valueListenable: preferences.vmDeveloperModeEnabled,
+                builder: (context, vmDeveloperModeEnabled, _) {
+                  if (!vmDeveloperModeEnabled) {
+                    return Container();
+                  }
+                  return Row(
+                    children: [
+                      ModeDropdown(widget.controller),
+                      const SizedBox(width: denseSpacing),
+                    ],
+                  );
+                },
+              ),
+              const SizedBox(width: denseSpacing),
```

Comment:
remove this sized box. If vm developer mode is not enabled, this would result in double padding

Reply:
Must have missed this one. Good catch!